cms/djangoapps/contentstore/docs/diagrams/certificate_available_date.dsl

```diff
@@ -1,6 +1,6 @@
 /*
- * This is a high level diagram visualizing how the `CERTIFICATE_AVAILBLE_DATE` and "visible date" attribute updates
```

Comment:
can I just state how much easier it made this documentation update that image was generated with DSL  input which was also next to it in the repository? 🙏 @justinhynes